modules/rostests/apitests/shell32/SHIsBadInterfacePtr.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for SHIsBadInterfacePtr
+ * COPYRIGHT:   Copyright 2025 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include "shelltest.h"
+#include <undocshell.h>
+
+typedef BOOL (WINAPI *FN_SHIsBadInterfacePtr)(LPCVOID, UINT_PTR);
+
+// FIXME: Kill me
```

Comment:
Will this be killed in the final version of the PR?

Reply:
If possible.